src/components/Table/Columns.tsx

```diff
@@ -172,7 +173,11 @@ export const allColumns: AllColumns = {
 	listedAt: {
 		header: 'Listed',
 		accessor: 'listedAt',
-		Cell: ({ value }) => <span style={{ whiteSpace: 'nowrap' }}>{value} days ago</span>
```

Comment:
we don't need `whiteSpace: 'nowrap'` since it's already set in the https://github.com/snqb/defillama-app/blob/a154e1fd12ca47b0a8dbbe42622d483a4b5b96f6/src/components/Table/shared.tsx#L36